download/linux.md

```diff
@@ -18,7 +18,9 @@ Settings can be transferred by copying `~/.local/share/Vorta/settings.db` to `~/
 ## Install from Source
 The most generic way is to install it as Python package using [PIP](https://pip.readthedocs.io/en/stable/installing/). First [install](https://borgbackup.readthedocs.io/en/stable/installation.html) Borg using the package of your distribution or via PyPI. The latter needs some additional [source packages](https://borgbackup.readthedocs.io/en/stable/installation.html#dependencies). Then install Vorta from PyPI. Your local Python version should be >= 3.6.
 ```
+$ pip3 install --upgrade setuptools pip
```

Comment:
If you use this outside your virtual environment, it will probably break stuff.
It could be included in a troubleshooting subsection though (same for export path)